Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -127,15 +130,46 @@ public static void Init(Assembly masterAssembly)
 			ResourceClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Id" || x.Name == "Resource_Id");
 			StyleClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Style" || x.Name == "Resource_Style");
 			LayoutClass = masterAssembly.GetTypes().FirstOrDefault(x => x.Name == "Layout" || x.Name == "Resource_Layout");
+
+			_resourceTypeNames.Add(DrawableClass, "drawable");
+			_resourceTypeNames.Add(ResourceClass, "id");
+			_resourceTypeNames.Add(StyleClass, "style");
+			_resourceTypeNames.Add(LayoutClass, "layout");
 		}
 
-		internal static int IdFromTitle(string title, Type type)
+		static int IdFromTitle(string title, Type type)
```

Comment:
Weird. Why did he add the internal? https://github.com/xamarin/Duplo/commit/7e971911eebffd2d2de516f86bafc4cd54b6a5d5#diff-6d2fedc564b28324406a6ff722fad0e9R53

Reply:
yea nothing uses it so I figured just remove it :-/